GitHubCore.ps1

```diff
@@ -703,6 +716,7 @@ function Invoke-GHRestMethodMultipleResult
                 'AccessToken' = $AccessToken
                 'TelemetryProperties' = $telemetryProperties
                 'TelemetryExceptionBucket' = $errorBucket
+                'ApiVersion' = $ApiVersion
```

Comment:
Let's keep this ordered the same way as the params:

```suggestion
                'ApiVersion' = $ApiVersion
                'AccessToken' = $AccessToken
                'TelemetryProperties' = $telemetryProperties
                'TelemetryExceptionBucket' = $errorBucket
```

Reply:
Oh shoot. I thought I did that, committed that change.